README.rst

```diff
@@ -43,7 +43,7 @@ To use this container, use these simple instructions:
 
 ::
 
-    docker run -d --name armada -p 8000:8000 -v ~/.kube/config:/root/.kube/config -v $(pwd)/examples/:/examples -v /tmp:/dev/log quay.io/attcomdev/armada:latest
+    docker run -d --name armada quay.io/attcomdev/armada:latest
```

Comment:
so this run does still need the volume mount for the kube config and the examples.